### PR TITLE
GEA-1736 Fix timestamp marshall on event

### PR DESCRIPTION
**Pull request by pangea-andrest**
- Fix timestamp mashall to work in the same way as server
- Little fix when unmarshall error occurs
- Little fix on test (avoid to check message, it should change)

### CODE REVIEW

**Review by dbenders — COMMENTED** [review]
I'm not sure if I like having our own timestamp with the only purpose of proper marshalling/unmarshalling... can't be pass a custom function to the marshaller as we do in python?

**Comment by pangea-andrest** [comment]
@dbenders I'm not sure about it, based on the examples I found, this is the way you have to marshall/unmarshall in a custom way

**Review by dbenders — APPROVED** [review]

File: service/audit/service.go [hunk 1]
```diff
@@ -21,13 +21,14 @@ type Audit struct {
...
-	VerifySignature bool
+	VerifyProofs          bool
+	SkipEventVerification bool
 }
 
 func New(cfg *pangea.Config, opts ...Option) (*Audit, error) {
 	cli := &Audit{
-		Client: pangea.NewClient("audit", cfg),
+		Client:                pangea.NewClient("audit", cfg),
+		SkipEventVerification: true,
```
Comment (dbenders) [inline service/audit/service.go:31]: true by default?

File: service/audit/api.go [hunk 2]
```diff
@@ -200,17 +207,17 @@ func SearchAllAndValidate(ctx context.Context, client Client, input *SearchInput
 }
 
 func (a *Audit) verifyRecords(events SearchEvents, root *Root) error {
-	for idx, event := range events {
-		if a.VerifyProofs && !event.VerifyHash() {
-			return fmt.Errorf("audit: cannot verify hash of record [%v]", idx)
+	for _, event := range events {
+		if a.SkipEventVerification && !event.VerifyHash() {
```
Comment (dbenders) [inline service/audit/api.go:211]: The meaning is switched (Skip means no verification)

File: service/audit/service.go (hunk 1 above)
Comment (pangea-andrest) [reply service/audit/service.go]: You are right, my bad. Fixed.

File: service/audit/api.go (hunk 2 above)
Comment (pangea-andrest) [reply service/audit/api.go]: You are right, my bad. Fixed.

**Review by dbenders — APPROVED** [review]

**Pull request merged**